blockchain/processing.py

```diff
@@ -307,17 +311,20 @@ def _execute_phase_2(self, config, phase_1_info):
 
             valid_transactions, invalid_transactions = self.check_tx_requirements(phase_1_info.transactions)
 
+            # TODO: take lower_phase_hash out of verification_info and have it hashed in sign_verification instead
```

Comment:
Yes, it's done. I will blow it away today. Weird, I thought I took the lower_phase_hash out of those phase messages. I will make sure they're out today.